lib/numbers.md

```diff
@@ -9,16 +9,16 @@ Fixnum for integers and Float for decimal numbers.
 
 # What are some common operations and comparisons you would perform on numbers?
 
-+, -, *, /, ** and % for addition, substraction, multiplication, division, exponential multiplication and modular division.
++, -, *, /, ** and % for addition, substraction, multiplication, division, exponential multiplication and modular division.  The most common comparisons are ==, !=, >, <, >= and <= for equals, does not equal, greater than, less than, greater than or equal to and less than or equal to.
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-The '+' operator produces the sum of two number, or concatenates two strings.
+The '+' operator produces the sum of two numbers, while it concatenates two strings.
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
 It will not work unless you convert it to another data type.  "20".to_i will convert the string to an integer.
```

Comment:
What result to you get when you run `"20" * 3`? Does this operation still _work_, even if it doesn't work the way we'd expect/hope it to?


If you do convert `"20"` to an Integer, then try to _divide_ it by 3, do you get a precise answer? How might you get a more precise answer?